ts/a11y/explorer/KeyExplorer.ts

```diff
@@ -188,16 +188,19 @@ export class SpeechExplorer extends AbstractKeyExplorer<string> {
    * @override
    */
   public Start() {
+    let options = this.getOptions();
+    if (SRE.engineSetup().locale !== options.locale) {
+      SRE.setupEngine({locale: options.locale});
+    }
```

Comment:
There is a potential problem, here, as `SRE` may not be available yet.  Importing `sre.js` may start an asynchronous load of SRE, and that may not have completed when this is called.  So you may need to do this in an `sreReady()` promise as well (or within an `srePromise.then()` call, where `srePromise` is imported from `sre.js`, and is resolved when SRE has been loaded, though perhaps is not yet ready).

Reply:
This is now in called only after the promise resolves. Then creates another promise, in case a new locale file needs to be loaded. This also fixes the problem of order for the `KeyExplorers` that we had discussed.